docker/demisto-sdk/README.md

```diff
@@ -0,0 +1,59 @@
+# Demisto-SDK docker
```

Comment:
we should document the env vars that need to be modified on `docker run` for some of the commands